template.html

```diff
@@ -0,0 +1,143 @@
+</div>
+
+<div id='missing_check_box' style='display:none;'>
+	<h2 id='missing_check_box_phrase' style='color:red;'>All questions are mandatory.</h2>
+</div>
+
+<div id='working_page'>
+	<h1 style="text-align:center;" id="statement">
+		
+	</h1>
+	<div id="document">
+		<div id="loading" class="alert alert-info">
+			<strong>Loading the PDF file...</strong>
+		</div>
+		<canvas id="the_canvas" style="border: 1px solid black; display:none;"/>
+	</div>
+
+	<div id="questions">
+	<div>
+</div>
+
+<script src="http://mozilla.github.com/pdf.js/build/pdf.js"></script>  
+<script>
+
+var scale = 1.5;
+var canvas = document.getElementById('the_canvas');
+var context = canvas.getContext('2d');
+
+//Not sure we should keep that worker in production
+PDFJS.workerSrc = 'http://mozilla.github.com/pdf.js/build/pdf.worker.js';
+
+function resetTask(){
+	$('#questions').empty();
+	context.clearRect ( 0 , 0 , canvas.width, canvas.height );
+}
+
+
+pybossa.taskLoaded(function(task, deferred) {
+	if ( !$.isEmptyObject(task) ) {
+
+		//Here's the code to get the PDF name from /upload/pdfs/ located at the pybossa root folder 
+		//and display it into the page
+		PDFJS.getDocument("../../../uploads/pdfs/" + task.info.pdf).then(function(pdf) {
+			pdf.getPage(task.info.page).then(function(page) {
+				task.p = page;
+				deferred.resolve(task);
+			})
+		});
+	} else {
+        	deferred.resolve(task);
+	}
+});
+
+pybossa.presentTask(function(task, deferred) {
+	// Code used to create the form
+	if ( !$.isEmptyObject(task) ) {
+		$('#statement').html(task.statement === undefined ? "This document refers to a particular European Comission public consultation. Please read it and  check the checkboxes at the end of the document to help us filing it." : task.statement);
+		$('#loading').hide();
+		$('#the_canvas').show();
+		var viewport = task.p.getViewport(scale);
+		canvas.height = viewport.height;
+		canvas.width = viewport.width;
+
+		var renderContext = {
+			canvasContext: context,
+			viewport: viewport
+		};
+		
+		// Generating the PDF on the canvas
+		task.p.render(renderContext);
+
+		var j = 1
+		
+		// Generating  the HTML
+		$('#questions').append("<p><form id='choices'></form></p>");
+		for(question in task.info.questions){
+			var answer_type = task.info.questions[question][0];
+                        $('#choices').append("<fieldset id='question" + 
+                                                j + 
+                                                "'> <legend>" + 
+                                                question + 
+                                                "</legend>" + 
+                                                 (answer_type === 'multiple' ? 'You can check <strong>several</strong> answers' : 'You can <strong>only</strong> check <strong>one</strong> answer') +
+                                                "<br></fieldset>");
```

Comment:
Indent function arguments to another line and check if it is still readable for the ternary operator part.